<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>309</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Steganography</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushcode">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>371</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Code</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushdecode">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>371</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Decode</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>394</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string> Input container file:     </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="push_input">
       <property name="text">
        <string>Open...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="minimumSize">
        <size>
         <width>170</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>392</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string> Input file for code:       </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="push_inputF">
       <property name="text">
        <string>Open...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_2">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="minimumSize">
        <size>
         <width>170</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>393</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string> Output container file:  </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="push_output">
       <property name="text">
        <string>Save...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_3">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="minimumSize">
        <size>
         <width>170</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>391</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string> Output file for decode:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="push_outputF">
       <property name="text">
        <string>Save...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_4">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_8">
       <property name="minimumSize">
        <size>
         <width>170</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
